executor/builder.go

```diff
@@ -167,7 +167,7 @@ func (b *executorBuilder) buildTableScan(v *plan.TableScan) Executor {
 		seekHandle:  math.MinInt64,
 	}
 	x := b.buildFilter(e, v.FilterConditions)
-	if v.Desc && !supportDesc {
```

Comment:
`supportDesc` represents if KV layer supports XAPI descending order. 
If we can't use XAPI, this variable should not be used.